docs/installation.rst

```diff
@@ -6,14 +6,12 @@ Installing Percona Toolkit
 
 Percona provides packages for most popular 64-bit Linux distributions:
 
-* Debian 7 ("wheezy")
-* Debian 8 ("jessie")
-* Ubuntu 14.04 LTS (Trusty Tahr)
-* Ubuntu 16.04 LTS (Xenial Xerus)
-* Ubuntu 16.10 (Yakkety Yak)
-* Ubuntu 17.04 (Zesty Zapus)
-* Red Hat Enterprise Linux or CentOS 6 (Santiago)
-* Red Hat Enterprise Linux or CentOS 7 (Maipo)
+* Debian 8 ("Jessie")
```

Comment:
https://www.percona.com/downloads/percona-toolkit/LATEST/  there is no Debian 8 choice...